azure-pipelines.yml

```diff
@@ -263,28 +263,13 @@ jobs:
 
 - job: execute_verilated_tests_englishbreakfast
   displayName: Execute tests on the Verilated English Breakfast toplevel design
-  pool: ci-public
-  dependsOn:
-    - chip_englishbreakfast_verilator
-    - sw_build_englishbreakfast
+  pool:
+    vmImage: ubuntu-18.04
   steps:
   - template: ci/install-package-dependencies.yml
-  - template: ci/download-artifacts-template.yml
-    parameters:
-      downloadPartialBuildBinFrom:
-        - chip_englishbreakfast_verilator
-        - sw_build_englishbreakfast
   - bash: |
-      # Install an additional pytest dependency for result upload.
-      pip3 install pytest-azurepipelines
-
       . util/build_consts.sh
-      pytest --version
-      pytest test/systemtest/englishbreakfast/test_sim_verilator.py \
-        -m "not slow" \
-        --log-cli-level=DEBUG \
-        --test-run-title="Run English Breakfast system tests with Verilator simulation" \
-        --napoleon-docstrings
+      ci/scripts/run-english-breakfast-verilator-tests.sh
```

Comment:
Why aren't you adding in this functionality first and deleting the old functionality once you're confident it's stable?

Reply:
Yeah I realized this right after I submitted -- I'll add it back and move my changes into a separate rule.